spec/requests/api/v0/post_manifest_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 describe 'POST manifests' do
+  before(:each) do
+    @current_session = mock_authenticated_session
```

Comment:
We need a user to associate with each manifest, so yes, authentication is required.

I will inline; the fact that I didn't indicates I didn't add enough tests for authz.